apps/client/src/styles/about.css

```diff
@@ -1,3 +1,3 @@
 html body {
-  background-color: #000;
+  background-color: #fff;
```

Comment:
if we want the background to be white on _all_ pages, we should probably put it into an `index.css` file that we import from index.tsx.